devTools/cardCreator/cards_to_import/MakantorWarbeast.minion.card.json

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "Makantor Warbeast",
+  "description": "Rush. After attacking, deal 4 damage to all surrounding enemies",
```

Comment:
add a new line after rush in the description
"Rush. \n After attacking, deal 4 damage to all surrounding enemies"